src/app/mat-main-nav/mobile-menu/mobile-menu.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit, ViewEncapsulation } from "@angular/core";
+import { menuItems } from "../menu-items.const";
+import { IMenuItem } from "../menu-item.interface";
+import { environment } from "@environment";
+
+@Component({
+  selector: "app-mobile-menu",
+  templateUrl: "./mobile-menu.component.html",
+  encapsulation: ViewEncapsulation.None
```

Comment:
just for my own understanding why is encapsulation set to none in this component?

Reply:
css overrides, considering either to decouple css and turn off view encapsulation because of w3c html validation by default. no attributes ng-content... for css scoping

Reply:
css overrides can still be achieved using the deep selector within the component's scss file. 

so are you saying the markup with encapsulation turned on wont pass w3c validation?

Reply:
Kinda, if directives (which are attributes) are not square bracketed they will fail. also angular adds its component host id for css view encapsulation so you'd get a lot of these...
![Screenshot 2020-03-16 at 10 32 44](https://user-images.githubusercontent.com/13540819/76748022-f9e94080-6771-11ea-8082-0841011f3973.png)

Reply:
turned off all view encapsulations as we'd probably need accessibility exercises at some point to rectify these. p.s. nhsuk-frontend riddled with errors as well on every page. e.g. https://validator.w3.org/nu/?doc=https%3A%2F%2Fservice-manual.nhs.uk%2F

Reply:
cheers, i think we ought to create a task for this and do some further investigation. would be good to get some clarity around if w3c compliant html is necessary for a single page app but if so perhaps this is something angular universal can help with